RealmSwift/Tests/ResultsTests.swift

```diff
@@ -186,7 +186,7 @@ class ResultsTests: TestCase {
         innerArray.array.append(SwiftObject())
         outerArray.array.append(innerArray)
         realm.add(outerArray)
-        XCTAssertEqual(Int(1), outerArray.array.filter("ANY array IN %@", realm.objects(SwiftObject)).count)
+        // FIXME: XCTAssertEqual(Int(1), outerArray.array.filter("ANY array IN %@", realm.objects(SwiftObject)).count)
```

Comment:
It passes for me. Did you delete the `build` directory first? The RealmSwift project never rebuilds Realm.framework if it already exists, and using a build from master will compile and fail a few tests.